src/app/common/TrayView.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <KSlideout
+    class="tray"
+    close-button-alignment="end"
+    :has-overlay="false"
+    is-visible
+    prevent-close-on-blur
```

Comment:
Did we say somewhere we not prevent close on blur? I think I at least asked the question, not sure what the outcome was. I'm gonna approve this anyway as its one of those things we can change later if need be, 2 tics

Reply:
> I'm pretty sure when I tried it without it didn't matter, I could click the rows and change the tray, plus blur and close the tray. Lets get this one in though, we can always change that later if necessary.

Hm, you’re right. I’m not sure what’s different now than when I added this. 🤔 

I removed it now as it behaves like you described.